src/main/java/org/jabref/model/entry/field/InternalField.java

```diff
@@ -10,16 +10,17 @@
  * JabRef internal fields
  */
 public enum InternalField implements Field {
-    INTERNAL_ALL_FIELD("all"),
-    INTERNAL_ALL_TEXT_FIELDS_FIELD("all-text-fields"),
-    MARKED_INTERNAL("__markedentry"),
+    MARKED_INTERNAL("__markedentry"), // used in old versions of JabRef. Currently used for conversion only
     OWNER("owner"),
     TIMESTAMP("timestamp", FieldProperty.DATE),
     GROUPS("groups"),
     KEY_FIELD("bibtexkey"),
     TYPE_HEADER("entrytype"),
     OBSOLETE_TYPE_HEADER("bibtextype"),
-    BIBTEX_STRING("__string"),
+    // all field names starting with "Jabref-internal-" are not appearing in .bib files
+    BIBTEX_STRING("JabRef-internal-bibtex-string"), // marker that the content is just a BibTeX string
```

Comment:
Changing these values might lead to problems as they are (maybe) stored in bib files as keys for save actions or in the preferences.

Reply:
I checked the usage of this string. It is called at the formatter only.

Reply:
Ok, but `all` and `all-text-field` should be used elsewhere too.

Reply:
I really thought on these things and they MUST NOT stored in `bib` files. Otherwise, something will go completely wrong.

Reply:
If I'm not mistaken, they are used in the serialization of save actions, e.g.
````
@Comment{
   jabref-meta: saveActions:enabled;
   all[normalize_month]
}
````

Reply:
Ups. OK, I put the old values back.